pkg/core/blockchain.go

```diff
@@ -1305,6 +1305,11 @@ func (bc *Blockchain) verifyHashAgainstScript(hash util.Uint160, witness *transa
 		return err
 	}
 
+	gasPolicy := bc.contracts.Policy.GetMaxVerificationGas(interopCtx.DAO)
```

Comment:
But we don't have it in the policy contract as of C# node. We'll need to make a separate proposal for it, but at the moment this would break storage contents compatibility.

Reply:
In fact, there is one already going that direction: #1794.

Reply:
Well this is only an interface: we can retrieve values via `Policy` without changing storage.